config/clusters/nasa-cryo/common.values.yaml

```
@@ -0,0 +1,122 @@
+basehub:
+  nfs:
+    pv:
+      # from https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html
+      mountOptions:
+        - rsize=1048576
+        - wsize=1048576
+        - timeo=600
+        - soft # We pick soft over hard, so NFS lockups don't lead to hung processes
+        - retrans=2
+        - noresvport
+      serverIP: fs-0872256335d483d5f.efs.us-west-2.amazonaws.com
+      baseShareName: /
+  jupyterhub:
+    custom:
+      2i2c:
+        add_staff_user_ids_to_admin_users: true
+        add_staff_user_ids_of_type: "github"
+      homepage:
+        templateVars:
+          org:
+            name: Cryo in the Cloud
+            logo_url: https://github.com/CryoInTheCloud/CryoCloudWebsite/blob/main/cryocloud.png
+            url: https://github.com/CryoInTheCloud
+          designed_by:
+            name: 2i2c
+            url: https://2i2c.org
+          operated_by:
+            name: 2i2c
+            url: https://2i2c.org
+          # Ideally, this community would like to list more than one funder
+          # Issue tracking implementation of this feature:
+          # https://github.com/2i2c-org/default-hub-homepage/issues/16
+          funded_by:
+            name: "NASA ICESat-2 Science Team"
+            url: https://icesat-2.gsfc.nasa.gov/science_definition_team
+    hub:
+      config:
+        Authenticator:
+          # This hub uses GitHub Teams auth and so we don't set
+          # allowed_users in order to not deny access to valid members of
+          # the listed teams. These people should have admin access though.
+          admin_users:
+            - tsnow03
+            - JessicaS11
+            - dfelikson
+        JupyterHub:
+          authenticator_class: github
+        GitHubOAuthenticator:
+          allowed_organizations:
+            - 2i2c-org:tech-team
+            - CryoInTheCloud:CryoCloudUser
+            - CryoInTheCloud:CryoCloudAdvanced
+    singleuser:
+      serviceAccountName: cloud-user-sa
+      defaultUrl: /lab
+      # User image repo: https://github.com/CryoInTheCloud/CryoCloudWebsite/tree/main/conda
+      image:
+        # This image is available on both Docker Hub and quay.io. We use quay.io
+        # here due to its more generous pull rate limits.
+        name: quay.io/cryointhecloud/cryocloudwebsite
+        tag: "2022.10.12"
+      storage:
+        extraVolumeMounts:
+          - name: home
+            mountPath: /home/jovyan/shared
+            subPath: _shared
+            readOnly: false
```

```suggestion
            readOnly: true
```

I think by default, we should make shared be readonly (and shared-readwrite available to admins as read-write). I think for some specific hub we made shared read-write for everyone, but don't think that should be default